bot.js

```diff
@@ -51,8 +51,10 @@ This bot demonstrates many of the core features of Botkit:
     -> http://howdy.ai/botkit
 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
-var env = require('node-env-file');
-env(__dirname + '/.env');
+if (process.env.NODE_ENV !== 'production') {
+  var env = require('node-env-file');
+  env(__dirname + '/.env');
+}
 
```

Comment:
This conditional logic would be unnecessary if `dotenv` were used in place of `node-env-file`

https://github.com/motdotla/dotenv


* `node-env-file` also appears to be broken, in the sense that there is unconventional behavior:  ENV values that are defined later do not override those defined earlier.